### PR TITLE
New UI for facebook accounts

### PR DESCRIPTION
**Pull request by lkazberova**

### CODE REVIEW

File: src/screens/Integrations/Facebook/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        <br />
+        <Small>This account cannot be connected at this time</Small>
+    </div>
+);
+function IntegrationsFacebookForm(props) {
+    const { open, classes, onClose, accounts, excluded, editAccount } = props;
+    const [enabledIds, enable] = useState(
+        accounts.filter(item => !item.disabled).map(item => item.id),
+    );
+    console.log(
```
Comment (juanplusjuan) [inline src/screens/Integrations/Facebook/Form.jsx]: @lkazberova can you remove the `console.log` instances please?  Also, I thought that Husky would prevent these from being committed in the first place.  Is there a reason that you're bypassing it?

File: src/state/ducks/account.js [hunk 2]
```diff
@@ -177,11 +202,29 @@ export const nonExistedRequiredAccountsSelector = createSelector(
...
+);
 /**
  * Action Creators
  * */
 
+export const openFacebookAccountsWindow = (accounts = [], excluded = []) => ({
+    type: SHOW_FACEBOOK_ACCOUNTS_WINDOW,
+    payload: { accounts, excluded },
+});
+export const closeFacebookAccountsWindow = () => ({
```
Comment (juanplusjuan) [inline src/state/ducks/account.js]: @lkazberova The first time we show the modal I think the "Cancel" action implies that the user has changed their mind about connecting the Facebook accounts.  In this case, I believe we should disable them all.  Any subsequent time that the user decides to edit the account then "Cancel" should just close the window.

File: src/state/ducks/account.js [hunk 3]
```diff
@@ -249,6 +292,11 @@ function* authorizeAccountSaga({ payload }) {
             type: AUTHORIZE_ACCOUNT.SUCCESS,
             payload: results,
         });
+        if (payload.type === "facebook") {
```
Comment (juanplusjuan) [inline src/state/ducks/account.js]: @lkazberova Can you make "facebook" a string constant that we reference from a single location?

File: src/screens/Integrations/Overview.jsx [hunk 4]
```diff
@@ -58,22 +61,27 @@ class IntegrationsOverview extends React.Component {
                 </Section>
                 <Grid item xs={12}>
                     {isLoadingFacebook && <LinearIndeterminateProgress />}
-                    {facebookAccounts.map(account => (
+                    {/* {facebookAccounts.map(account => ( */}
```
Comment (juanplusjuan) [inline src/screens/Integrations/Overview.jsx]: @lkazberova Should we remove this commented section?

**Comment by lkazberova** [comment]
@juanplusjuan I suppose I resolve all issues, please take a look, thank you!

**Pull request merged**